ember-cli-build.js

```diff
@@ -1,28 +1,18 @@
 'use strict';
 
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
-// eslint-disable-next-line node/no-extraneous-require
-const semver = require('semver');
-const fs = require('fs');
 
 module.exports = function(defaults) {
+  let app = new EmberAddon(defaults, {
+    // Add options here
+  });
+
```

Comment:
I _think_ we can add in our package.json here [like so](https://guides.emberjs.com/release/addons-and-dependencies/#toc_anonymous-amd-javascript-modules) to import the version for our `clientInfo` helper.